### ISSUE TITLE
Tags whitelist doesn't seem to work

### ISSUE THREAD

**Issue by tekkitan** [zendesklabs/ticket_field_manager_app#13]
I am trying to test hiding certain fields by using tags, but it doesn't seem to be working. It works by group just fine, but if I put the tag in the whitelist and add the tag to the ticket it does not unhide the fields.

**Comment by jstjoe** [comment]
Hey @tekkitan I think the whitelisting by tag is for **user tags**, not ticket tags. @luizfaias can you confirm that?

**Comment by luizfaias** [comment]
@jstjoe you are right. All the whitelist settings are based on the current agent (this.currentUser().id()).

The verification happens here: https://github.com/zendesklabs/ticket_field_manager_app/blob/master/app.js#L121
